templates/partials/modal_messagehistory.haml

```diff
@@ -7,7 +7,7 @@
     .message-history
       .action.stackitem{ ng-repeat:"action in actions" }
         .action-time
-          [[ action.created_on | autodate ]]
+          <cp-tooltip display-text="[[ action.created_on | autodate]]" tooltip-text="[[ action.created_on | fulldate ]]" position="top-right"><cp-tooltip>
```

Comment:
Wondering whether we want to be using the filters `autodate` and `fulldate` in the directive itself, or have them used when we give things to the directive (as we do at the moment). Both seem reasonable, just thought it was worth mentioning.